src/plugins/data_source_management/public/components/data_source_menu/data_source_selectable.test.tsx

```diff
@@ -54,10 +58,28 @@ describe('DataSourceSelectable', () => {
     );
     expect(component).toMatchSnapshot();
     expect(client.find).toBeCalledWith({
-      fields: ['id', 'description', 'title'],
+      fields: ['id', 'title', 'auth.type'],
       perPage: 10000,
       type: 'data-source',
     });
     expect(toasts.addWarning).toBeCalledTimes(0);
   });
+
+  it('should filter options if configured', async () => {
```

Comment:
Where we configure this filterfn, is configurable in the yml file or somewhere or must change the code to configure the filter? thanks

Reply:
This filterFn is passed by plugins when they consume the data source menu component, it is used then in the component when it is mounted to remove unwanted data sources before rendering